### DISCUSSION TITLE
CoPilot User Activity

### DISCUSSION THREAD

**Discussion by cnc-bdibene** [microsoft/PowerShellForGitHub#419] [General]
Is it possible to generate the report that is in the GUI at https://github.com/organizations/cast-and-crew/settings/copilot/seat_management:
github.com/organizations/%YourOrgName% > Settings > CoPilot > Access > Get Report

It lists all users assigned a CoPilot license and a datestamp of Last Activity, which is what I'm specifically after.

**Comment by HowardWolosky** [comment]
It does appear to be possible with this API that's currently in beta:
https://docs.github.com/en/rest/copilot/copilot-for-business?apiVersion=2022-11-28

For API's that aren't already directly implemented in the module, it's always possible to manually use them with `Invoke-GHRestMethod`.  You may want to give that a try and see if you get the results that you're looking for.

More info in the wiki: https://github.com/microsoft/PowerShellForGitHub/wiki/Invoke-GHRestMethod

**Reply by cnc-bdibene** [reply]
yeah, I've already tried with the new API.  It sort of works.  I'll give Invoke-GHRestMethod a try. Thanks.

**Reply by cnc-bdibene** [reply]
FYI, GitHub Support advised it is not possible to get the info via API.
They created a feature request for this.
Thanks for your help.
